tests/pkcli_data/command_info.in/example_pkcli_module.py

```diff
@@ -4,4 +4,4 @@
 def _example_pkcli_function():
     pykern.pkcli.command_info("this is a message")
 
-_example_pkcli_function()
\ No newline at end of file
+_example_pkcli_function()
```

Comment:
Need a test for check_eof_newline().

Reply:
a44f8b2fb1e292d1140765110bbb79627445f1f2